docs/src/examples/gallery.md

```diff
@@ -296,7 +296,7 @@ savefigs("spline-quadratic", ans) # hide
         mesh,
         scatter,
         samples = 10,
-        domain = 0:1
+        domain = "0:1"
```

Comment:
I wish we could fix this to work in general, could be done later.

Reply:
Made a reminder in #74.